en/rocky/8/guides/add_mirror_manager.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Adding a public mirror to Rocky's mirror manager'
+---
+
+## What You Need
+* An account on https://accounts.rockylinux.org/
+
+# Adding a public mirror to Rocky's mirror manager
+
+Rocky uses Fedora's Mirror Manager for organizing community mirrors.
+
+## Creating a site
+
+Access Rocky's Mirror Manager here: https://mirrors.rockylinux.org/mirrormanager/
+
+After a successful login, your profile will be on the top right. Select the drop down then click "My sites".
```

Comment:
Note that if you've enabled 2-factor authentication on your rocky account, put both the normal account password and the OTP code concatenated together in the password field to login.